### PR TITLE
mavlink: send mwosd-extra compatible modes

### LINKED ISSUE #1770: Flight Mode missing from MAVLink protocol?
Apparently flight mode status is not included in the MAVLink output, so OSDs using MAVLink protocol, specifically minimOSD-Extra are unable to display current flight mode.

### PR DESCRIPTION
**Pull request by mlyle**
Attempts to resolve #1770 .  Needs test.

### CODE REVIEW

File: flight/Libraries/mavlink/v1.0/common/custom_types.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+#define CUSTOM_MODE_STAB 0
+#define CUSTOM_MODE_ACRO 1
+#define CUSTOM_MODE_ALTH 2
+#define CUSTOM_MODE_AUTO 3
+#define CUSTOM_MODE_GUIDED 4
+#define CUSTOM_MODE_LOITER 5
+#define CUSTOM_MODE_RTL 6	/* Return to "launch" */
+#define CUSTOM_MODE_CIRCLE 7
+#define CUSTOM_MODE_POSH 8	/* so posh. */
```
Comment (peabody124) [inline flight/Libraries/mavlink/v1.0/common/custom_types.h]: now you are being silly :-p

**Comment by peabody124** [comment]
Code looks good, even with silly comments. Good to merge once tested.

**Comment by mlyle** [comment]
;) the only reason I do this is to litter things with snarky comments

**Comment by OhmEye** [comment]
Tested with minimOSD-extra and works for me. Thanks!

**Comment by mlyle** [comment]
Cool, thanks everyone!

**Pull request merged**